charts/posthog/values.yaml

```diff
@@ -592,6 +594,29 @@ clickhouse:
     shardsCount: 1
     replicasCount: 1
 
+  ## -- ClickHouse settings configuration.
+  ## You can use this to override settings, for example `prometheus/port: 9363`
+  ## For the full list of settings, see:
+  ## - https://clickhouse.com/docs/en/operations/settings/settings/
+  settings: {}
+    # Uncomment those lines if you want to enable the built-in Prometheus HTTP endpoint in ClickHouse.
+    # prometheus/endpoint: /metrics
+    # prometheus/port: 9363
+    # prometheus/metrics: true
+    # prometheus/events: true
+    # prometheus/asynchronous_metrics: true
+
+  ## -- Default settings configuration for ClickHouse. Don't overwrite this!
+  defaultSettings:
+    format_schema_path: /etc/clickhouse-server/config.d/
```

Comment:
Will this change interfere with existing deployments in any way?

Reply:
I've only moved it from being hardcoded to a variable. The rendered template didn't change so I think the answer is no.